pkg/probe/log_fortianalyzer_queue.go

```diff
@@ -0,0 +1,55 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type LogAnaQueueResults struct {
+	Connected  int `json:"connected"`
+	FailedLogs int `json:"failed_logs"`
+	CachedLogs int `json:"cached_logs"`
+}
+
+type LogAnaQueue struct {
+	Results LogAnaQueueResults `json:"results"`
+	VDOM    string             `json:"vdom"`
+	Version string             `json:"version"`
+}
+
+func probeLogAnalyzerQueue(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		logAnaConn = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_queue_connected",
+			"Fortianalyzer queue connected state",
+			[]string{"vdom"}, nil,
+		)
+		logAnaFail = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_queue_failed",
+			"Failed logs in fortianalyzer queue",
+			[]string{"vdom"}, nil,
+		)
+		logAnaCach = prometheus.NewDesc(
+			"fortigate_log_fortianalyzer_queue_cached",
+			"Cached logs in fortianalyzer queue",
+			[]string{"vdom"}, nil,
+		)
```

Comment:
```suggestion
		logAnaLogs = prometheus.NewDesc(
			"fortigate_log_fortianalyzer_queue_logs",
			"State of logs in the queue",
			[]string{"vdom", "state"}, nil,
		)
```
Use labels instead of extra metrics